### PR TITLE
ENH: Install `tractolearn` from GitHub

### PR DESCRIPTION
**Pull request by jhlegarreta**
Install `tractolearn` from GitHub

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@felixdumais have not tested.

File: Dockerfile [hunk 1]
```diff
@@ -52,7 +41,21 @@ RUN rm -rf /var/lib/apt/lists/* && \
         | tar -xz -C /opt/ants-2.3.4 --strip-components 1
 
 RUN pip install -U pip
-RUN cd /tractolearn && pip install -e . 
+
+RUN pip install git+https://github.com/scil-vital/tractolearn.git@{version} tractolearn
```
Comment (jhlegarreta) [inline Dockerfile]: The version should be pinned once it is tagged in `tractolearn`.

Comment (fdumais) [reply Dockerfile:45]: see PR #4

**Comment by jhlegarreta** [comment]
@fdumais So removing the additions and working directory makes the trick?

**Comment by fdumais** [comment]
Yes, can you just remove the `workdir` and all the `add` and replace with this:
`RUN git clone https://github.com/scil-vital/tractolearn.git /tractolearn`

**Comment by jhlegarreta** [comment]
> Yes, can you just remove the workdir and all the add and replace with this:
RUN git clone https://github.com/scil-vital/tractolearn.git /tractolearn

Done. In the mid-term we will be willing to pin the version so that a stable/tagged/release version is used.

**Comment by jhlegarreta** [comment]
Unrelated to this patch set:
https://github.com/scil-vital/dockerfile_tractolearn/blob/main/Dockerfile#L57

The directory change in there is redundant as there is another one changing to the same directory immediately before.

File: Dockerfile (hunk 1 above)
Comment (jhlegarreta) [reply Dockerfile]: Tracked in #5. Resolving.

**Pull request merged**